webapp/src/pages/Settings.tsx

```diff
@@ -548,8 +569,40 @@ const Settings: React.FC = () => {
     </FormGroup>
   );
 
+  const displayAnalysesCustomizationGeneralSection = () => (
+    <Box display="flex" justifyContent="flex-start" gap={5} alignItems="center">
+      <FormControl variant="standard" sx={{ m: 1, width: 120 }}>
+        <InputLabel id="supported-language-input-label">
+          supported_language
```

Comment:
As you can see 3 lines below, the field is named `language` and not `supported_language`. The list of the languages we support can be called `SUPPORTED_LANGUAGES`, that's perfect, but not the one language selected from the list. Does that make sense?
```suggestion
          language
```

Reply:
Okay. Make sense. I will apply the suggestion